src/views/dashboard.js

```diff
@@ -100,13 +108,13 @@ export const Dashboard = (props) => {
                 <h2 className="page-title">Admin Dashboard</h2>
                 <div className="dashboard__modules_container">
                     <DashboardModule
-                        data={MODULE_DATA.openTickets}
+                        data={widgetData.openTickets}
```

Comment:
API is returning "opentickets" instead of "openTickets" so just gotta fix the casing on it and it should work!